packages/webdriverio/src/constants.ts

```diff
@@ -320,3 +321,70 @@ export const ERROR_REASON = [
     'ConnectionFailed', 'NameNotResolved', 'InternetDisconnected',
     'AddressUnreachable', 'BlockedByClient', 'BlockedByResponse'
 ]
+
+/**
+ * Special Characters
+ */
+export const Key = {
+    /**
+     * Special control key that works cross browser for Mac, where it's the command key, and for
+     * Windows or Linux, where it is the control key.
+     */
+    Ctrl: 'WDIO_CONTROL',
+    NULL: UNICODE_CHARACTERS.NULL,
+    Cancel: UNICODE_CHARACTERS.Cancel,
+    Help: UNICODE_CHARACTERS.Help,
+    Backspace: UNICODE_CHARACTERS.Backspace,
+    Tab: UNICODE_CHARACTERS.Tab,
+    Clear: UNICODE_CHARACTERS.Clear,
+    Return: UNICODE_CHARACTERS.Return,
+    Enter: UNICODE_CHARACTERS.Enter,
+    Shift: UNICODE_CHARACTERS.Shift,
+    Control: UNICODE_CHARACTERS.Control,
+    Alt: UNICODE_CHARACTERS.Alt,
+    Pause: UNICODE_CHARACTERS.Pause,
+    Escape: UNICODE_CHARACTERS.Escape,
+    Space: UNICODE_CHARACTERS.Space,
+    PageUp: UNICODE_CHARACTERS.PageUp,
+    PageDown: UNICODE_CHARACTERS.PageDown,
+    End: UNICODE_CHARACTERS.End,
+    Home: UNICODE_CHARACTERS.Home,
+    ArrowLeft: UNICODE_CHARACTERS.ArrowLeft,
+    ArrowUp: UNICODE_CHARACTERS.ArrowUp,
+    ArrowRight: UNICODE_CHARACTERS.ArrowRight,
+    ArrowDown: UNICODE_CHARACTERS.ArrowDown,
+    Insert: UNICODE_CHARACTERS.Insert,
+    Delete: UNICODE_CHARACTERS.Delete,
+    Semicolon: UNICODE_CHARACTERS.Semicolon,
+    Equals: UNICODE_CHARACTERS.Equals,
+    Numpad0: UNICODE_CHARACTERS['Numpad 0'],
+    Numpad1: UNICODE_CHARACTERS['Numpad 1'],
+    Numpad2: UNICODE_CHARACTERS['Numpad 2'],
+    Numpad3: UNICODE_CHARACTERS['Numpad 3'],
+    Numpad4: UNICODE_CHARACTERS['Numpad 4'],
+    Numpad5: UNICODE_CHARACTERS['Numpad 5'],
+    Numpad6: UNICODE_CHARACTERS['Numpad 6'],
+    Numpad7: UNICODE_CHARACTERS['Numpad 7'],
+    Numpad8: UNICODE_CHARACTERS['Numpad 8'],
+    Numpad9: UNICODE_CHARACTERS['Numpad 9'],
+    Multiply: UNICODE_CHARACTERS.Multiply,
+    Add: UNICODE_CHARACTERS.Add,
+    Separator: UNICODE_CHARACTERS.Separator,
+    Subtract: UNICODE_CHARACTERS.Subtract,
+    Decimal: UNICODE_CHARACTERS.Decimal,
+    Divide: UNICODE_CHARACTERS.Divide,
+    F1: UNICODE_CHARACTERS.F1,
+    F2: UNICODE_CHARACTERS.F2,
+    F3: UNICODE_CHARACTERS.F3,
+    F4: UNICODE_CHARACTERS.F4,
+    F5: UNICODE_CHARACTERS.F5,
+    F6: UNICODE_CHARACTERS.F6,
+    F7: UNICODE_CHARACTERS.F7,
+    F8: UNICODE_CHARACTERS.F8,
+    F9: UNICODE_CHARACTERS.F9,
+    F10: UNICODE_CHARACTERS.F10,
+    F11: UNICODE_CHARACTERS.F11,
+    F12: UNICODE_CHARACTERS.F12,
+    Command: UNICODE_CHARACTERS.Command,
```

Comment:
In the w3c specs this key is called the `Meta` key. There's also a differentiation between left and right but I'm not sure if that matters, since it's in the spec I think it was worthy to mention both :)

Reference: https://www.w3.org/TR/uievents-code/#key-alphanumeric-functional

Reply:
Interesting! The [WebDriver spec](https://w3c.github.io/webdriver/webdriver-spec.html#keyboard-actions) defines two different values for `Meta` which I assume is the one for left and right. I will revisit these tables to see if they were misinterpreted or got updated.